sites/site-utilities/src/components/logo/index.tsx

```diff
@@ -17,6 +16,12 @@ const backgroundStyle = {
     alignItems: "center",
     justifyContent: "space-between",
     padding: "8px",
+    backgroundColor: "var(--neutral-fill-rest)",
+};
+const headerStyle = {
+    fontSize: "14px",
+    fontWeight: 600,
+    lineHeight: "20px",
```

Comment:
Curious if we want to map to font weight and line height variables from the design system here

Reply:
No problem. Done.